frontend/class-opengraph.php

```diff
@@ -576,7 +576,14 @@ public function image( $image = false ) {
 		$opengraph_images = new WPSEO_OpenGraph_Image( $this->options, $image );
 
 		foreach ( $opengraph_images->get_images() as $img ) {
-			$this->og_tag( 'og:image', esc_url( $img ) );
+			$og_image_tag = 'og:image';
+
+			if (0 === strpos($img, 'https://'))
```

Comment:
You can setup the codestyle locally
```
composer install
composer config-yoastcs
```

You've to do these two steps one and you can run the codestyle check by doing this command: `vendor/bin/phpcs`